x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -173,6 +176,118 @@ export function relatedEventsByEntityId(data: DataState): Map<string, ResolverRe
   return data.relatedEvents;
 }
 
+/**
+ * Seed word/non-word boundaries with zero-width spaces so they break/wrap nicely
+ *
+ * @param potentiallyLongString A string to seed with zero-width spaces so it wraps nicely
+ */
+function addWordBreakSpaces(potentiallyLongString: string): string {
+  // Seed boundaries beetween word and non-word characters with Zero-width spaces to allow for text to be wrapped when it overflow its container.
+
+  return potentiallyLongString.replace(/(\w\W)|(\W\w)/g, (boundary: string) => {
+    return `${boundary[0]}\u200b${boundary[1]}`;
```

Comment:
:grey_question: @oatkiller I ended up pulling this into the selector here. Is this OK?